app/components/Views/confirmations/components/smart-account-update-splash/smart-account-update-splash.tsx

```diff
@@ -68,7 +78,16 @@ export const SmartAccountUpdateSplash = () => {
     onReject(serializedError as unknown as Error);
   }, [onReject]);
 
-  if (!transactionMetadata || acknowledged) {
+  const onConfirm = useCallback(() => {
+    dispatch(upgradeSplashPageAcknowledgedForAccount(from));
+    setAcknowledged(true);
+  }, [dispatch, from, setAcknowledged]);
+
+  if (
+    !transactionMetadata ||
+    acknowledged ||
+    (from && upgradeSplashPageAcknowledgedForAccounts.includes(from))
```

Comment:
```suggestion
    upgradeSplashPageAcknowledgedForAccounts.includes(from)
```

`from` is always settled as `transactionMetadata` already goes into short circuit above

Reply:
This is to fix ts complain.